OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/UpdateSubscriptionOperation.kt

```diff
@@ -81,20 +81,30 @@ class UpdateSubscriptionOperation() : Operation(SubscriptionOperationExecutor.UP
             setEnumProperty(::status.name, value)
         }
 
+    /**
+     * The jwt token used for the operation that updates a subscription.
+     */
+    var jwt: String?
+        get() = getStringProperty(::jwt.name)
+        private set(value) {
+            setStringProperty(::jwt.name, value!!)
```

Comment:
This `!!` is dangerous, since the constructor for this class just below says `jwt` passed in can be `null`, and it will call this set method directly within it.